### PR TITLE
ITSI-8389: Update SFX python client to support additional params

### PR DESCRIPTION
**Pull request by achandrashekhar-splunk**
These parameters are required for optimizing some tasks on signalfx and to return additional metadata (custom properties/tags etc) to the callers.

### CODE REVIEW

**Comment by mpetazzoni** [comment]
@achandrashekhar-splunk Keep in mind that this is a public, open-source repository. I would avoid referencing internal Jira issue numbers in the PR title and commit message.

**Comment by achandrashekhar-splunk** [comment]
@mpetazzoni noted :) I've edited the PR description

**Comment by mahdibh** [comment]
This commit has broken our python client - see https://signalfuse.atlassian.net/browse/SWAT-1915 for the full details. @tedoc2000  has already sent out a PR for a fix that we merged.

@achandrashekhar-splunk can you clarify why these extra fields are required? Right now we don't have any of these fields documented and they are untested (for example, the adjustableResolution which this PR made the default is completely broken right now).

File: signalfx/signalflow/computation.py [hunk 1]
```diff
@@ -165,8 +165,10 @@ def stream(self):
             if isinstance(message, messages.InfoMessage):
                 self._process_info_message(message.message)
                 self._batch_count_detected = True
-                if self._current_batch_message:
-                    yield self._get_batch_to_yield()
+                yield message
+                # TODO: InfoMessages are coming before DataMessages
+                # if self._current_batch_message:
+                #     yield self._get_batch_to_yield()
```
Comment (mahdibh) [inline signalfx/signalflow/computation.py:171]: is this an intended change ?

**Comment by achandrashekhar-splunk** [comment]
CCing Syam here, for context

**Pull request merged**